qiskit_experiments/data_processing/processor_library.py

```diff
@@ -49,8 +56,12 @@ def get_processor(
     meas_return = run_options.get("meas_return", MeasReturnType.AVERAGE)
     normalize = analysis_options.get("normalization", True)
 
+    num_qubits = experiment_data.metadata["num_qubits"]
```

Comment:
Should this be
```suggestion
    num_qubits = experiment_data.metadata.get("num_qubits", 1)
```
? Are we guarenteeded to have `num_qubits` in the metadata?